CONTRIBUTING.md

```diff
@@ -99,6 +99,9 @@ Commits that affect all packages or are not related to any (e.g. changes to NPM
 Package releases are made using Lerna's release capabilities and executed by [the technical committee](https://github.com/webdriverio/webdriverio/blob/master/GOVERNANCE.md#the-technical-committee) only. To run it just call:
 
 ```sh
+# ensure to have pulled the latest code
+$ git pull origin master
```

Comment:
This should probably be "git pull upstream master" since we are asking people to fork right?

Reply:
I don't think it is important how you've called your remote end. This is less a step by step description for newbies in the project and more an information for someone in the technical committee to make sure to pull the latest master. Everyone in the TC probably made enough contribution that he already started . pushing to `origin` per default.